src/basic/linux/if_ether.h

```diff
@@ -81,7 +81,7 @@
 #define ETH_P_ATMMPOA	0x884c		/* MultiProtocol Over ATM	*/
 #define ETH_P_PPP_DISC	0x8863		/* PPPoE discovery messages     */
 #define ETH_P_PPP_SES	0x8864		/* PPPoE session messages	*/
-#define ETH_P_LINK_CTL	0x886c		/* HPNA, wlan link local tunnel */
+#define ETH_P_LINK_CTL	0x886c		/* HPNA, wlan link-local tunnel */
```

Comment:
Please do not edit this file. This is imported from the kernel source.

Reply:
Many thanks for reviewing. Corrected.